Nodejs/Product/Nodejs/SharedProject/CommonProjectNode.cs

```diff
@@ -858,26 +790,20 @@ private void TriggerIdle()
 
         internal void CreateSymLinkWatcher(string curDir)
         {
-            if (!CommonUtils.HasEndSeparator(curDir))
-            {
-                curDir = curDir + Path.DirectorySeparatorChar;
-            }
-            this.symlinkWatchers[curDir] = CreateFileSystemWatcher(curDir);
+            curDir = CommonUtils.EnsureEndSeparator(curDir);
+            this.fileWatcher.ObserveFolder(curDir);
         }
 
         internal bool TryDeactivateSymLinkWatcher(HierarchyNode child)
```

Comment:
>HierarchyNode child [](start = 50, length = 19)

It's strange that this isn't symmetric with `CreateSymLinkWatcher`, but I see that that's not new.